mivs/templates/mivs_judging/index.html

```diff
@@ -16,7 +16,7 @@
 <tbody>
 {% for game in games %}
     <tr>
```

Comment:
quick thing: I want to avoid us using c.URL_BASE at all costs because it breaks some configurations.  if there's a way to specify this link as a relative link, let's do that.  that isn't possible in emails, but, should be possible on all non-email templates.

see https://github.com/magfest/ubersystem/issues/1534 for more info on this topic